Dockerfile

```diff
@@ -46,7 +46,16 @@ RUN case ${TARGETPLATFORM} in \
 
 # Copy binary to runtime image
 FROM ${RUNTIME_IMAGE}
+ARG VERSION
+
 COPY --from=builder /go/src/github.com/oauth2-proxy/oauth2-proxy/oauth2-proxy /bin/oauth2-proxy
 COPY --from=builder /go/src/github.com/oauth2-proxy/oauth2-proxy/jwt_signing_key.pem /etc/ssl/private/jwt_signing_key.pem
 
+LABEL org.opencontainers.image.licenses=MIT \
+      org.opencontainers.image.description="A reverse proxy that provides authentication with Google, Azure, OpenID Connect and many more identity providers." \
+      org.opencontainers.image.source=https://github.com/oauth2-proxy/oauth2-proxy \
+      org.opencontainers.image.url=https://github.com/oauth2-proxy/oauth2-proxy \
```

Comment:
```suggestion
      org.opencontainers.image.url=https://quay.io/oauth2-proxy/oauth2-proxy \
```

Reply:
```org.opencontainers.image.url URL to find more information on the image (string)```
shouldn't that be the github readme file and stuff?

Reply:
I read it as `image.url`. The url from which to retrieve the image.

`image.source` is the url to the source code hosted on github

`image.documentation` could directly point to the docs

Reply:
Renovate (which is the one I care about) only looks at [source](https://docs.renovatebot.com/modules/datasource/docker/) so i'm good with that update